src/foam/lib/csv/CSVSupport.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package foam.lib.csv;
+
+import java.io.BufferedReader;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import foam.core.ClassInfo;
+import foam.core.PropertyInfo;
+import foam.core.FObject;
+import foam.lib.parse.*;
+import foam.dao.Sink;
+
+
+public class CSVSupport 
+  extends foam.core.ContextAwareSupport
+{
+  protected Parser headParser = new Parser() {
```

Comment:
You can just do headParser = new Repeat(new CSVSTtringParser(), new Literal(","));

No need for the anonymous class if all it does is call it's delegate.